README.md

```diff
@@ -21,6 +21,17 @@ Last updated on November 1st, 2023
 
 **Note** on GPT4 Turbo. While the above figures show it to be comparable to GPT4, this is due to us filtering out some documents that some of the models refuse to summarize. When comparing to GPT 4 on all summaries (both GPT4 models summarize all documents) the turbo model is around 0.3% worse than GPT4, but still better than GPT 3.5 Turbo.
 
+<table>
+  <tr>
+    <td style="text-align: center; vertical-align: middle;">
+      <img src="img/candle.png" width="50" height="50">
```

Comment:
The page will load much faster if you scale the image down to 50x50 ahead of time. The candle image is pretty big.